pom.xml

```diff
@@ -162,8 +157,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.2</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
Maybe we can include this information as a major change for 1.9 RELEASE in a changelog?